sdcm/sct_config.py

```diff
@@ -2183,7 +2183,9 @@ def _check_backend_defaults(self, backend, required_params):
     def _check_version_supplied(self, backend: str):
         options_must_exist = []
 
-        if not self.get('use_preinstalled_scylla') and not backend == 'baremetal':
+        if (not self.get('use_preinstalled_scylla') and
+                not backend == 'baremetal' and
+                not self.get('unified_package')):
```

Comment:
Not critical, but in such cases we should move out the `not` out of parentheses and flip the `and/or` operators among operands:

```
if not (self.get('use_preinstalled_scylla') and backend == 'baremetal' and self.get('unified_package')):
```